esmf_regrid/tests/unit/experimental/io/test_round_tripping.py

```diff
@@ -110,38 +121,57 @@ def _make_mesh_to_grid_regridder(
         use_src_mask = False
         use_tgt_mask = False
 
-    rg = MeshToGridESMFRegridder(
+    kwargs = {
+        "mdtol": 0.5,
+        "tgt_resolution": resolution,
+        "use_src_mask": use_src_mask,
+        "use_tgt_mask": use_tgt_mask,
+    }
+    if regridder == MeshToGridESMFRegridder:
+        kwargs["method"] = method
+    rg = regridder(
         src,
         tgt,
-        method=method,
-        mdtol=0.5,
-        tgt_resolution=resolution,
-        use_src_mask=use_src_mask,
-        use_tgt_mask=use_tgt_mask,
+        **kwargs,
     )
     return rg, src
 
 
+def _compare_ignoring_var_names(x, y):
+    old_var_name = x.var_name
+    x.var_name = y.var_name
```

Comment:
Is this actually good enough -- what about the var-names of coordinates, or the mesh (for instance) ?

Obviously though, it does work for now.

Reply:
I think the idea is that this is for comparing coordinates, which would only have the surface level var name. comparing meshes is a bit awkward currently and so is a bit of a blind spot across iris_esmf_regrid (as noted in various code comments). perhaps this can be clarified in the naming of this helper function?

Reply:
OK I didn't quite grasp why the coordinate names might not also be changed in the save process,
but if you're happy then it doesn't really matter.